mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -497,7 +499,9 @@ impl From<TendermintCoinRpcError> for ValidatePaymentError {
             TendermintCoinRpcError::PerformError(e) | TendermintCoinRpcError::RpcClientError(e) => {
                 ValidatePaymentError::Transport(e)
             },
-            TendermintCoinRpcError::InternalError(e) => ValidatePaymentError::InternalError(e),
+            TendermintCoinRpcError::InternalError(e) | TendermintCoinRpcError::NotFound(e) => {
+                ValidatePaymentError::InternalError(e)
+            },
```

Comment:
Here `TendermintCoinRpcError::NotFound` is matched to `InternalError`, but in other places to `Transport`